### PR TITLE
[ITSASU-926] Improve coverage by removing overly complex code

### PR DESCRIPTION
**Pull request by aug24**

### CODE REVIEW

File: app/connectors/utilities/ConnectorUtils.scala [hunk 1]
```diff
@@ -24,17 +24,17 @@ trait ConnectorUtils[L <: ErrorResponsesModel, R] extends JsonUtils {
 
   type Response = Either[ErrorModel, R]
 
-  def parseFailure(status: Int, jsValue: JsValue)(implicit lReader: Reads[L]): Response = {
+  def parseFailure(status: Int, jsValue: JsValue)(implicit lReader: Reads[L]): Response = Left (
```
Comment (sb-dev) [inline app/connectors/utilities/ConnectorUtils.scala:27]: I am not sure about this part, why did you add the `Left`?

Comment (aug24) [reply app/connectors/utilities/ConnectorUtils.scala:27]: It was always there - it was just added magically by an implicit.  I have removed the implicit because it really was making the codebase more complex, not less (and the implicit file wasn't well covered, which is the original point of the ticket).

**Review by sb-dev — APPROVED** [review]
